nsqd/stats_test.go

```diff
@@ -32,7 +32,7 @@ func TestStats(t *testing.T) {
 	identify(t, conn, nil, frameTypeResponse)
 	sub(t, conn, topicName, "ch")
 
-	stats := nsqd.GetStats()
+	stats := nsqd.GetStats(topicName, "ch")
```

Comment:
it's probably a good idea to expand this test a bit - put a message to one other topic and channel

Reply:
Sounds reasonable. Done.